cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaAvailabilityTest.java

```diff
@@ -274,17 +262,17 @@ Admin ac() {
     @Test
     public void testBelowMinIsr(VertxTestContext context) {
         KSB ksb = new KSB()
-            .addNewTopic("A", false)
-                .addToConfig(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG, "2")
-                .addNewPartition(0)
+            .addNewTopic("A")
+                .addToConfig("2")
+                .addNewPartition()
```

Comment:
Are you sure the partition number will not be needed in the future for example?

Reply:
As above, will revert.